README.md

```diff
@@ -110,6 +110,10 @@ const handleCheckbox = async (id, e) => {
 - We recommend [enabling console persistence](https://stackoverflow.com/questions/5327955/how-to-make-google-chrome-javascript-console-persistent) while developing replicache-enabled apps to make debugging easier.
 - Remember that data changes can happen "underneath" you and cause `subscribe()` to re-fire at any time. These changes can come from the server or from a different tab. If your UI is not reactive (driven solely by the data model) you need to take extra steps to ensure the UI is in sync with the data.
 
+### Errors
+
+Critical errors that prevent Replicache from working are always thrown. We also use `console.error` / `console.warn` for errors that does not prevent Replicache from working but might temporary show confusing data until the next sync (this can happen if a _mutator_ is missing for example). Console errors/warnings should be take action on either by you or us.
```

Comment:
"Critical errors that prevent Replicache from working are always thrown." -- this feels a bit vague to me. what does "critical" mean? does my application have to stop or can i continue trying to call into replicache? maybe we can make it more specific, either saying which errors or types of errors are fatal vs which there is some hope of continuing.